### PR TITLE
fix(queries): align descriptionText to similar queries across different platforms #2

### PR DESCRIPTION
**Pull request by roi-orca**
align descriptionText to similar queries across different platforms #2

**Proposed Changes**
- Align descriptionText to similar queries across different platforms

1. API Gateway Without SSL Certificate
2. Automatic Minor Upgrades Disabled
3. CloudTrail Multi Region Disabled
4. DB Instance Publicly Accessible
5. ELB Using Insecure Protocols
6. Redshift Not Encrypted
7. Redshift Publicly Accessible
8. S3 Bucket With All Permissions
9. Node Auto Upgrade Disabled
10. API Gateway With Invalid Compression
11. CloudFront Logging Disabled
12. ELB Using Weak Ciphers
13. RDS Storage Not Encrypted
14. Service Account Name Undefined Or Empty
15. Cloud Storage Bucket Versioning Disabled
16. DNSSEC Using RSASHA1

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by kaplanlior** [comment]
Follow up to #5446 (tagging for future reference).

Thanks @roi-orca 
Please also include the common criteria for the queries in each PR (just so we can differentiate them over time).

**Comment by kaplanlior** [comment]
A list of queries would help differentiate between PRs (without looking at the files themselves).
e.g.

Aligned description of similar quires:
- API Gateway With Invalid Compression
- API Gateway Without SSL Certificate
- Automatic Minor Upgrades Disabled

**Comment by roi-orca** [comment]
@rafaela-soares can you approve please? :)

File: assets/queries/ansible/aws/automatic_minor_upgrades_disabled/metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "queryName": "Automatic Minor Upgrades Disabled",
   "severity": "LOW",
   "category": "Best Practices",
-  "descriptionText": "RDS instance auto minor version upgrade feature must be true",
+  "descriptionText": "RDS instance should have automatic minor upgrades enabled",
```
Comment (rafaela-soares) [inline assets/queries/ansible/aws/automatic_minor_upgrades_disabled/metadata.json:6]:
```suggestion
  "descriptionText": "RDS instance should have automatic minor upgrades enabled, which means the attribute 'auto_minor_version_upgrade' must be set to true.",
```

File: assets/queries/terraform/aws/automatic_minor_upgrades_disabled/metadata.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "queryName": "Automatic Minor Upgrades Disabled",
   "severity": "LOW",
   "category": "Best Practices",
-  "descriptionText": "RDS Instance Auto Minor Version Upgrade feature in Aws Db Instance must be true",
+  "descriptionText": "RDS instance should have automatic minor upgrades enabled",
```
Comment (rafaela-soares) [inline assets/queries/terraform/aws/automatic_minor_upgrades_disabled/metadata.json:6]:
```suggestion
  "descriptionText": "RDS instance should have automatic minor upgrades enabled, which means the attribute 'auto_minor_version_upgrade' must be set to true.",
```

File: assets/queries/cloudFormation/aws/cloudfront_logging_disabled/metadata.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "queryName": "CloudFront Logging Disabled",
   "severity": "MEDIUM",
   "category": "Observability",
-  "descriptionText": "Make sure AWS CloudFront distribution has access log enabled",
+  "descriptionText": "AWS Cloudfront distributions must have logging enabled",
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/cloudfront_logging_disabled/metadata.json:6]:
```suggestion
  "descriptionText": "AWS Cloudfront distributions must have logging enabled, which means the attribute 'DistributionConfig.Logging' must be defined",
```

File: assets/queries/ansible/aws/cloudtrail_multi_region_disabled/metadata.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "queryName": "CloudTrail Multi Region Disabled",
   "severity": "MEDIUM",
   "category": "Observability",
-  "descriptionText": "Check if MultiRegion is Enabled",
+  "descriptionText": "CloudTrail multi region should be enabled",
```
Comment (rafaela-soares) [inline assets/queries/ansible/aws/cloudtrail_multi_region_disabled/metadata.json:6]:
```suggestion
  "descriptionText": "CloudTrail multi region should be enabled, which means attribute 'is_multi_region_trail' should be set to true",
```

File: assets/queries/ansible/aws/redshift_not_encrypted/metadata.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "queryName": "Redshift Not Encrypted",
   "severity": "HIGH",
   "category": "Encryption",
-  "descriptionText": "Check if 'encrypted' field is false or undefined (default is false)",
+  "descriptionText": "AWS Redshift Cluster should be encrypted. Check if 'encrypted' field is false or undefined (default is false)",
```
Comment (rafaela-soares) [inline assets/queries/ansible/aws/redshift_not_encrypted/metadata.json:6]: Can you also update for `kics\assets\queries\cloudFormation\aws\redshift_not_encrypted\metadata.json`, please?

File: assets/queries/cloudFormation/aws/api_gateway_with_invalid_compression/metadata.json [hunk 6]
```diff
@@ -3,7 +3,7 @@
   "queryName": "API Gateway With Invalid Compression",
   "severity": "MEDIUM",
   "category": "Encryption",
-  "descriptionText": "Attribute 'MinimumCompressionSize' should be set and it's value should be greater than -1 and smaller than 10485760.",
+  "descriptionText": "API Gateway should have valid compression, which means attribute 'MinimumCompressionSize' should be set and it's value should be greater than -1 and smaller than 10485760.",
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/api_gateway_with_invalid_compression/metadata.json:6]:
```suggestion
  "descriptionText": "API Gateway should have valid compression, which means attribute 'MinimumCompressionSize' should be set and its value should be greater than -1 and smaller than 10485760.",
```

File: assets/queries/terraform/aws/api_gateway_with_invalid_compression/metadata.json [hunk 7]
```diff
@@ -3,7 +3,7 @@
   "queryName": "API Gateway With Invalid Compression",
   "severity": "MEDIUM",
   "category": "Encryption",
-  "descriptionText": "Attribute 'minimum_compression_size' should be set and it's value should be greater than -1 and smaller than 10485760.",
+  "descriptionText": "API Gateway should have valid compression, which means attribute 'minimum_compression_size' should be set and it's value should be greater than -1 and smaller than 10485760.",
```
Comment (rafaela-soares) [inline assets/queries/terraform/aws/api_gateway_with_invalid_compression/metadata.json:6]:
```suggestion
  "descriptionText": "API Gateway should have valid compression, which means attribute 'minimum_compression_size' should be set and its value should be greater than -1 and smaller than 10485760.",
```

File: assets/queries/cloudFormation/aws/rds_storage_not_encrypted/metadata.json [hunk 8]
```diff
@@ -3,7 +3,7 @@
   "queryName": "RDS Storage Not Encrypted",
   "severity": "HIGH",
   "category": "Encryption",
-  "descriptionText": "AWS RDS DB Instance should be encrypted",
+  "descriptionText": "RDS Storage should be encrypted",
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/rds_storage_not_encrypted/metadata.json:6]:
```suggestion
  "descriptionText": "RDS Storage should be encrypted, which means the attribute 'StorageEncrypted' should be set to 'true'",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello, @roi-orca!

Thank you so much for continuing to improve the descriptions 🚀

I only have a few suggestions. Let me know what you think about them, please 😊

**Comment by roi-orca** [comment]
@rafaela-soares resolved comments 👍

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Comment by roi-orca** [comment]
@rafaela-soares how I can re-run the check that failed?
`Warning:  uploadChunk (start: 100663296, end: 134217727) failed: Cache service responded with 503`
https://github.com/Checkmarx/kics/runs/6859677716?check_suite_focus=true

**Comment by rafaela-soares** [comment]
Hello, @roi-orca!

Yes, of course! The new release probably will be this wednesday😊

**Pull request merged**